src/extraction/fathom/ruleset_factory.js

```diff
@@ -337,12 +344,12 @@ export default class RulesetFactory {
 /**
  * Scale a number to the range [ZEROISH, ONEISH].
  *
- * For a rising trapezoid, the result is ZEROISH until the input reaches
+ * For a rising line, the result is ZEROISH until the input reaches
  * zeroAt, then increases linearly until oneAt, at which it becomes ONEISH. To
- * make a falling trapezoid, where the result is ONEISH to the left and ZEROISH
+ * make a falling line, where the result is ONEISH to the left and ZEROISH
  * to the right, use a zeroAt greater than oneAt.
  */
-function trapezoid(number, zeroAt, oneAt) {
+function linearScale(number, zeroAt, oneAt) {
```

Comment:
An optional change would be moving `zeroAt` and `oneAt` to named arguments via an object (i.e. simulating keyword arguments) so that call sites always name the parameters. That helps with the ambiguity of the slope changing based on the parameters.

Reply:
I think I'll leave it alone if it's all the same to you. The direction this function would expand in, if any, is to allow for more specified points (and more different slopes between them), and that would be impossible with named args.